packages/ariakit-test/src/__tests__/blur-test.tsx

```diff
@@ -3,7 +3,9 @@ import { blur } from "../blur.ts";
 import { q, render } from "../react.tsx";
 import { useAllEvents } from "./use-all-events.ts";
 
-test("blur", async () => {
+test("blur", async ({ skip }) => {
+  if (loader !== "react") skip();
+
```

Comment:
I believe we can make this test framework-agnostic, but we'll take care of that later.